app/models/ems_refresh/save_inventory_cloud.rb

```diff
@@ -171,7 +171,6 @@ def save_key_pairs_inventory(ems, hashes, target = nil)
   def save_cloud_volumes_inventory(ems, hashes, target = nil)
     target = ems if target.nil?
 
-    return unless ems.cloud_volumes
```

Comment:
@hsong-rh This should be OK the way it is. These methods aren't even called unless there's something to save in the hashes. OpenStack will no longer have cloud_volumes, etc in the hashes its parser generates, so these methods won't be called for OpenStack.